clmm/polaraveraging.py

```diff
@@ -159,14 +162,14 @@ def compute_tangential_and_cross_components(cluster=None,
 
```

Comment:
Suggestion re: ensuring all inputs are converted to arrays, since we probably have to do this a lot with astropy tables, maybe we should make a function in `utils.py` to do this check for all inputs in the first line of `_compute_*_shear()`, something like this:
```
def force_arrs(*args):  
    outargs = []  
    for arg in args:  
        outargs.append(np.array(arg))  
    return(outargs)
```

Reply:
Right, this is another way to approach this but given the small cope of this particular issue, I went with Camille's suggestion, removed the misleading `_list` from the variable name and updated the documentation stating that these should be arrays, not array-like. Letting the user take care of this, rather than making things more complex in the code.